docs/scos/dev/front-end-development/202307.0/oryx/oryx-localization.md

```diff
@@ -0,0 +1,148 @@
+
+## Auto-conversion of translation keys
+
+If a translation key doesn't match any of the translations, or if the i18n feature is not installed (which is the default behavior), the translation key is converted into a human-readable message.
+
+For example, if a token contains multiple parts, like `cart.increase`, you can provide a global translation for just the `increase` part or for `cart.increase`. This mechanism allows for a single translation of `increase` that might affect multiple components. This provides a consistent and convenient translation mechanism while remaining flexible to add specific localizations for some components.
+
+The following examples show how the tokens are translated.
+
+| TOKEN                       | CONVERTED LABEL |
+| --------------------------- | --------------- |
+| `cart.increase`             | Increase        |
+| `cart.add-to-cart`          | Add to cart     |
+| `cart.totals.<count>-items` | 5 items         |
+
+This mechanism lets Oryx avoid distributing localizations as a standard package or as part of the boilerplate. Labels are quite opinionated and may change frequently over time, which may cause breaking changes. For 90% of the cases, the keys are fairly short and provide an OK experience.
+
+{% info_block infoBox "" %}
+
+The [Oryx labs package](https://www.npmjs.com/package/@spryker-oryx/labs) provides some localizations mainly for demonstration reasons. For more information on the labs package, see [Feature sets](/docs/scos/dev/front-end-development/{{page.version}}/oryx/feature-setes.html).
+
+{% endinfo_block %}
+
+
+
+## Installing the i18n package
+
+To use the i18n package, add `I18nFeature` to the app and configure translation resources using the loader function:
+
+```ts
+import { appBuilder } from "@spryker-oryx/application";
+import { I18nFeature } from "@spryker-oryx/i18n";
+
+export const app = appBuilder()
+  .withFeature(
+    new I18nFeature({
+      // Here as example we are loading translations from TS modules
+      load: (localeId) => import(`../i18n/${localeId}.ts`), // <-- Required part
+    })
+  )
+  .create();
+```
+
+This gives you full flexibility of how and where to load your translation texts from. The `load` function returns a promise of a `{ default: I18nData }` type.
+
+## Translation resources
+
+To translate text in Oryx, you can provide translation data in a TS module.
+
+```ts {% raw %}
+export default {
+  increase: "Increase",
+
+  // using token parameters
+  "cart.entry.<quantity>-items": "x {quantity}",
+
+  // pluralization example
+  "order.<count>-items":
+    "Products ({count, plural, one {{count} item} other {{count} items}})",
+};
+{% endraw %}```

```

Comment:
not sure if it will be an issue in published docs, maybe not,.but in GitHub markdown preview it breaks rendering (code block is not closed)

Reply:
I tested the updated version, it will look as expected